samza-test/src/main/java/org/apache/samza/test/framework/system/CollectionStreamSystemSpec.java

```diff
@@ -32,11 +33,14 @@
  * Following system level configs are set by default
  * <ol>
  *   <li>"systems.%s.default.stream.samza.offset.default" = "oldest"</li>
```

Comment:
Please remove this line, if you end up removing this config in code (refer the comment on code below)